### ISSUE TITLE
issues calling next fn on scroll to top action

### ISSUE THREAD

**Issue by francisBae** [ankeetmaini/react-infinite-scroll-component#335]
Hello,
I'm having some issues using react-infinite-scroll-component.
according to the document, next function is 'a function which must be called after reaching the bottom'.
In my case, when I mouse scroll an reach the top, it still calls the next function.
I saw some code examples on codesandbox but couldn't find difference.
- is it related to the css styles?
does anybody have the same issue?

**Comment by muhammetakgun** [comment]
Yes, I have same issue, inverse={false} is not working. When I scroll mouse to the top it calls my service again. Does anybody here to help us?

**Comment by subu873** [comment]
Please watch this video for this library
https://youtu.be/c0nKjMnDfG4

**Comment by sanchitos** [comment]
I wrote an answer on stackoverflow. If there's a better solution please let me know.

https://stackoverflow.com/questions/73280173/how-to-avoid-scrollbar-staying-on-top-when-loading-more-data-infinite-scroll-i/74484560#74484560
